### PR TITLE
raw and with client apps

### PR DESCRIPTION
**Pull request by lukasz-glen**
nothing to add, just please check my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,19 @@ A curated list of precompiled Wasm binaries of
 programs that are known to successfully work with
 [Wasm sandbox](https://github.com/golemfactory/sp-wasm) in
 [Golem](https://github.com/golemfactory/golem).
+The list includes applications directly located in this repo,
```
Comment (kubkon) [inline README.md]:
```suggestion
The list includes applications located directly in this repo,
```

File: README.md [hunk 2]
```diff
@@ -3,10 +3,19 @@ A curated list of precompiled Wasm binaries of
...
+as well as links that points to external.
```
Comment (kubkon) [inline README.md]:
```suggestion
as well as links that point to external sources.
```

File: README.md [hunk 3]
```diff
@@ -3,10 +3,19 @@ A curated list of precompiled Wasm binaries of
...
+The applications can be raw or augmented with a client.
```
Comment (kubkon) [inline README.md]:
```suggestion
The applications can either be in a raw, Wasm format, or (preferably) they can be augmented with a GUI/CLI for the user's convenience.
```

File: README.md [hunk 4]
```diff
@@ -3,10 +3,19 @@ A curated list of precompiled Wasm binaries of
...
+When using raw, a user connects directly with Golem.
```
Comment (kubkon) [inline README.md]:
```suggestion
Using raw Wasm binaries implies that the user has to be able to prepare the corresponding `task.json` and the required folder structure themselves, and be able to directly connect with their Golem client (e.g., via the use of the [Golem CLI](https://github.com/golemfactory/golem-client)). 
```

File: README.md [hunk 5]
```diff
@@ -3,10 +3,19 @@ A curated list of precompiled Wasm binaries of
 programs that are known to successfully work with
 [Wasm sandbox](https://github.com/golemfactory/sp-wasm) in
 [Golem](https://github.com/golemfactory/golem).
+The list includes applications directly located in this repo,
+as well as links that points to external.
+The applications can be raw or augmented with a client.
+When using raw, a user connects directly with Golem. 
+That is more technical approach.
```
Comment (kubkon) [inline README.md]:
```suggestion
Therefore, as such, this approach requires some technical knowledge of the Golem's internals.
```

File: README.md [hunk 6]
```diff
@@ -3,10 +3,19 @@ A curated list of precompiled Wasm binaries of
...
 See [here](https://docs.golem.network/#/About/Use-Cases?id=wasm)
 to learn how to launch a Wasm task in Golem.
+The applications augmented with a client are more user friendly,
```
Comment (kubkon) [inline README.md]:
```suggestion
The applications augmented with a GUI/CLI are naturally more user friendly,
```

File: README.md [hunk 7]
```diff
@@ -3,10 +3,19 @@ A curated list of precompiled Wasm binaries of
...
+because it handles communication with Golem node,
```
Comment (kubkon) [inline README.md]:
```suggestion
because they handle communication with Golem node,
```

File: README.md [hunk 8]
```diff
@@ -3,10 +3,19 @@ A curated list of precompiled Wasm binaries of
...
+but there is no generic client. See g-flite for instance.
```
Comment (kubkon) [inline README.md]:
```suggestion
Having said that, there currently is no generic way of preparing such a GUI/CLI. There are some examples however. See the [g-flite](https://github.com/golemfactory/g-flite) app for instance.
```

File: README.md [hunk 9]
```diff
@@ -3,10 +3,19 @@ A curated list of precompiled Wasm binaries of
...
 
 The list:
+* [g-flite](https://github.com/golemfactory/g-flite) - text-to-speech
```
Comment (kubkon) [inline README.md:18]: @lukasz-glen How about we create 2 lists then: 1) with raw WASM binaries, and 2) with GUI/CLI client apps such as `g-flite`?

File: README.md [hunk 10]
```diff
@@ -25,8 +34,12 @@ $ git lfs pull
 ## Contributing
 We welcome contributions in the form of links to precompiled Wasm binaries of
 other programs. If you would like to submit such a link, do not hesitate to open a new PR.
+Your repo should contain README file and license. 
+If it is raw application, it should follow the below guidelines.
```
Comment (kubkon) [inline README.md]:
```suggestion
If it is a raw Wasm binary, it should follow the guidelines below.
```

File: README.md [hunk 11]
```diff
@@ -25,8 +34,12 @@ $ git lfs pull
...
+Approach to applications augemented with clients is more liberal.
```
Comment (kubkon) [inline README.md]:
```suggestion
For apps augmented with GUI/CLI, the requirements are more relaxed and not set in stone, with the only must-have: good user experience :-)
```

File: README.md [hunk 12]
```diff
@@ -25,8 +34,12 @@ $ git lfs pull
...
 
-When contributing, please make sure the repo the link will be pointing to adheres to the
+## Directories structure
+When contributing an application in raw flavor, please make sure the repo the link will be pointing to adheres to the
```
Comment (kubkon) [inline README.md]:
```suggestion
When contributing an application in a raw Wasm format, please make sure that the submitted link adheres to the
```

**Review by kubkon — CHANGES_REQUESTED** [review]

File: README.md (hunk 9 above)
Comment (lukasz-glen) [reply README.md:18]: sounds good

**Review by kubkon — APPROVED** [review]
LGTM!

**Pull request merged**